cmd/root.go

```diff
@@ -25,3 +33,27 @@ func Execute() error {
 func RootCmd() *cobra.Command {
 	return rootCmd
 }
+
+// CreateCommand creates a new command
+func CreateCommand(service KoolService, values NewCommandData) *cobra.Command {
```

Comment:
func CreateCommand(service KoolService, values NewCommandData) (newCmd *cobra.Command) {

Reply:
we will have different needs for other flags/parameters depending on the command... which makes me think that maybe `NewCommandData` is not worth.